Content.Server/Fluids/Components/BucketComponent.cs

```diff
@@ -70,11 +79,7 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
                 return false;
             }
 
-            if (mopComponent.CurrentVolume == mopComponent.MaxVolume)
-            {
-                Owner.PopupMessage(eventArgs.User, Loc.GetString("bucket-component-mop-is-full-message"));
```

Comment:
I removed the error message when a full mop clicks on the bucket, because I want it to have different behaviour when that happens (transfer from mop to bucket)